src/media/AvatarMember.jsx

```diff
@@ -14,22 +14,17 @@ export const AVATAR_PERSON_NOPHOTO_CLASS = 'avatar--noPhoto';
  */
 class AvatarMember extends React.PureComponent {
 	render() {
-		const {
-			member,
-			org,
-			fbFriend,
-			className,
-			...other
-		} = this.props;
+		const { member, org, fbFriend, className, small, ...other } = this.props;
 
-		const showNoPhoto = (member.photo || {}).photo_link == undefined;
+		const photoLink = small ? 'thumb_link' : 'photo_link';
```

Comment:
I feel like we should make the default the small one as this it the most likely way it will be used.

Reply:
@eilinora when should the `photo_link` be used then? On `large` versions?

Reply:
Yeah there are a couple of sizes... so large and above :)

Reply:
Hmm yeah that's probably a good idea. I'll wait for @mperrotti to be back to run this by him since it's a sorta small but still breaking change.

Reply:
@nlaz - All we're doing is asking for the smaller image asset for smaller avatars?
If so, LGTM :)